test/test_ad7768_1.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+hardware = "ad7768-1"
+classname = "adi.ad7768_1"
+
+
+#########################################
+@pytest.mark.iio_hardware(hardware)
+@pytest.mark.parametrize("classname", [(classname)])
+def test_ad7768_1_rx_data(test_dma_rx, iio_uri, classname, channel):
```

Comment:
Remove the "channel" argument

Reply:
I think I missed out to add channel parameter. This device has 1 channel, so I think we would need channel argument there.